skorch/helper.py

```diff
@@ -508,3 +509,83 @@ def describe_signature(self, df):
             )
 
         return signature
+
+
+class AccelerateMixin:
+    """Mixin class to add support for huggingface's accelerate
+
+    *Experimental*
```

Comment:
We can include a link to https://huggingface.co/docs/accelerate/index.html and how to install in the docstring

Reply:
If we agree to proceed with this design, I would add links and more information.